modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/VerticalCard.js

```diff
@@ -0,0 +1,145 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {ClayCardWithInfo} from '@clayui/card';
+import ClayIcon from '@clayui/icon';
+import ClaySticker from '@clayui/sticker';
+import React, {useState} from 'react';
+
+function getDataAttributes(data) {
```

Comment:
@carloslancha @markocikos once #449 is in upstream and I can rebase, I'll put this is it's own file.